### DISCUSSION TITLE
Which role is allowed to manage object categories & definitions ?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#252] [Documentation] [answered]
Suppose, we want to make changes in object categories & definitions, then can any role make changes in the object categories & definitions? Or is there any specific role who can manage the object categories & definitions changes?

cc: @rajeevsathish @reshmi-nair @Krishnaj20

**Comment by sharathkashyap** [comment, marked as answer]
@nidhikachhawa24 Kindly look into the https://knowlg.sunbird.org/learn/product-and-developer-guide/taxonomy-and-tagging which explains the significance of Object Category, Taxonomy in the system. Since these are tenant and root level settings which are done at t requires the user to be have "ADMIN" role.

cc: @rajeevsathish @Krishnaj20

**Reply by chitranshu-keshav** [reply]
@nidhikachhawa24 Can you please mark this thread as answered if your query is resolved?
